sogs/routes/rooms.py

```diff
@@ -337,3 +339,53 @@ def poll_room_info(room, info_updated):
             result['global_admin'] = True
 
     return jsonify(result)
+
+
+@rooms.post("/room/<Room:room>/file")
+@auth.user_required
+def upload_file(room):
+    """ upload a file to a room """
```

Comment:
Needs to be rebased on dev and the docs moved out of api.yaml to here (updated if needed).